BookPlayer/Library/ItemListViewController.swift

```diff
@@ -521,11 +574,12 @@ extension ItemListViewController: UITableViewDelegate {
     }
 
     func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
-        self.bulkControls.moveButton.isEnabled = true
-        self.bulkControls.trashButton.isEnabled = true
+        self.updateSelectionStatus()
     }
 
     func tableView(_ tableView: UITableView, didDeselectRowAt indexPath: IndexPath) {
+        self.updateSelectionStatus()
+
         guard tableView.indexPathForSelectedRow == nil else {
```

Comment:
we could leave only `self.updateSelectionStatus()` in this function (`didDeselectRowAt`) since the flags are being set in `self.updateSelectionStatus `